### DISCUSSION TITLE
NPM dependency error on web, not happening locally

### DISCUSSION THREAD

**Discussion by mattfelten** [codesandbox/codesandbox-client#5172] [Q&A]
I'm getting an error with one of our projects being imported into CodeSandbox. An error appears about `react-transition-group` which has [a number](https://github.com/react-bootstrap/dom-helpers/issues/51) [of issues](https://github.com/reactjs/react-transition-group/issues/556) around.

My issue is that I pulled a CodeSandbox down, and it runs completely fine locally. `npm start` and `npm run build` work perfectly. I turned it into a GitHub repo and imported back into CodeSandbox, the error appears again. I have no idea what's different about the project besides it being in the CodeSandbox client.

CodeSandbox: https://codesandbox.io/s/github/mattfelten/2dde3
GitHub Repo: https://github.com/mattfelten/2dde3

Any ideas?

**Comment by SaraVieira** [comment]
Hey!

This seems to be a real issue in our bundler, mind making an issue instead?

**Reply by mattfelten** [reply]
Made https://github.com/codesandbox/codesandbox-client/issues/5177, thanks!
